exer_4.py

```diff
@@ -0,0 +1,46 @@
+number= int(input("Enter the Number :"))
+
+# fibonacci series in recursive menner
+
+def fibo(n):
+    if n <=1:
+        return 1
+    return fibo(n-1) + fibo(n-2)
+
+for i in range(number):
+    print(fibo(i))
+
+
+#  fibonacci series in memoization menner
+
+cache  ={0:0,1:1}
```

Comment:
Format it properly, the space must be one before and after `=`.
`cache = {0: 0, 1: 1}`